evol/evolution.py

```diff
@@ -159,6 +160,17 @@ def mutate(self, func, probability=1.0, name=None, **kwargs) -> 'Evolution':
         """
         return self._add_step(MutateStep(name=name, probability=probability, func=func, **kwargs))
 
+    def log(self, name=None) -> 'Evolution':
+        """
+        Logs a population. If a Population object was initialized with a logger
+        object then you may specify how logging is handled. The base logging 
+        operation just logs to standard out. 
+
+        :return: self
+        """
+        return self._add_step(LogStep(name))
```

Comment:
I think we should allow kwargs here to let the user tell the logger _what_ to log. You may have multiple log steps and decide to log different things at each step.